app/Nova/TravelAssignment.php

```diff
@@ -199,9 +200,8 @@ static function (string $message) use ($validator, &$businessPolicyPassed): void
             if ($trip->fee_amount / $total_cost < config('travelpolicy.minimum_trip_fee_cost_ratio')) {
                 $validator->errors()->add(
                     'matrix_itinerary',
-                    'Trip fee must be at least '.
-                    (config('travelpolicy.minimum_trip_fee_cost_ratio') * 100).
-                    '% of the per-person cost for this trip.'
+                    'The airfare cost exceeds the amount allowed for this trip. Increase the trip fee to at '.
+                    'least 20% of the per-person total cost for this trip.'
```

Comment:
is this intentionally hard-coded as `20%`?

Reply:
Good catch